libr/bin/bfile.c

```diff
@@ -324,13 +324,13 @@ R_API RBinFile *r_bin_file_new(RBin *bin, const char *file, const ut8 *bytes, ut
 	}
 	int res = r_bin_file_set_bytes (binfile, bytes, sz, steal_ptr);
 	if (!res && steal_ptr) { // we own the ptr, free on error
-		free((void*) bytes);
+		free ((void *)bytes);
 	}
 	binfile->rbin = bin;
-	binfile->file = file? strdup (file): NULL;
+	binfile->file = file ? strdup (file) : NULL;
```

Comment:
nope. spacing goes after the ?, not before, isnt?

Reply:
I think we have both cases in our codebase... I was always used to ` ? ` and clang-format supports that.